src/Bridges/bridgeoptimizer.jl

```diff
@@ -239,12 +239,25 @@ end
 function MOI.supports(b::AbstractBridgeOptimizer,
                       attr::MOI.AbstractConstraintAttribute,
                       IndexType::Type{<:MOI.Index})
-    return MOI.supports(b.model, attr, IndexType)
+    if is_bridged(b, IndexType)
+        return MOI.supports(b.bridged, attr, IndexType)
+    else
+        return MOI.supports(b.model, attr, IndexType)
+    end
 end
+
 function MOI.set(b::AbstractBridgeOptimizer,
                  attr::MOI.AbstractConstraintAttribute,
                  index::MOI.Index, value)
```

Comment:
Use `::MOI.ConstraintIndex` to avoid method ambiguity

Reply:
Done